components/brave_wallet/browser/keyring_service_unittest.cc

```diff
@@ -1990,19 +1999,78 @@ TEST_F(KeyringServiceUnitTest, SelectHardwareAccount) {
   ASSERT_EQ(service.GetSelectedWalletAccount(), imported.front());
 }
 
-TEST_F(KeyringServiceUnitTest, SetSelectedAccount) {
+TEST_F(KeyringServiceUnitTest, SetSelectedAccount_CardanoEnabled) {
+  base::test::ScopedFeatureList feature_list{
+      features::kBraveWalletCardanoFeature};
```

Comment:
please try to always enable/disable features in a test fixture class to avoid any inconsistences during setup/teardown.